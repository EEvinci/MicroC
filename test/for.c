void main() {
  int n=5;
  int m=2;
  int i;
  int j;
  for (i = 0; i < n; i=i+1)
  {
    for(j = 0; j < m; j++)
      print i;
  }
}
