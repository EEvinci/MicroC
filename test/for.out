24 19 0 5 25 13 0 5 12 13 0 1 1 0 2 12 15 1 15 1 13 0 2 1 0 0 12 15 -1 16 100 13 0 3 1 0 0 12 15 -1 16 72 13 0 2 1 11 22 15 -1 0 1 0 1 1 15 -1 13 0 3 1 13 0 3 1 11 0 1 1 12 15 -1 13 0 3 1 11 13 0 1 1 11 7 18 42 13 0 2 1 13 0 2 1 11 0 1 1 12 15 -1 13 0 2 1 11 13 11 7 18 31 21 3